voidcrew/code/modules/projectiles/guns/energy/pulse.dm

```diff
@@ -1,9 +1,10 @@
 /obj/item/gun/energy/pulse/terra
 	name = "SolGov pulse rifle"
 	desc = "An almost jury-rigged answer to NT's dominance on pulse weaponry. Features only 4 shots but can rapidly recharge."
+	icon = 'voidcrew/icons/obj/guns/energy.dmi'
 	icon_state = "terrapulse"
 	ammo_type = list(/obj/item/ammo_casing/energy/laser/pulse)
-	internal_cell = TRUE //prevents you from giving it an OP cell - WS Edit
+	internal_cell = TRUE
```

Comment:
WTF

Reply:
this isn't related to the PR its just that... internal_cell is some stupid WS thing intended to let officers powergame their gun's power cell... i'll just remove it in a separate PR

Reply:
thats awesome